app/assets/javascripts/initializers/initializeArticleReactions.js

```diff
@@ -132,39 +103,74 @@ function reactToArticle(articleId, reaction) {
 
   getCsrfToken()
     .then(sendFetch('reaction-creation', createFormdata()))
-    .then(function(response) {
+    .then(function getCsrfTokenCallback(response) {
```

Comment:
you can switch to arrow functions if you don't want to invent names for inline callbacks and errbacks